src/components/symptom-shark/view/LogEntryEditView/LogEntryEditView.stories.tsx

```diff
@@ -0,0 +1,15 @@
+
+import React from "react";
+import SymptomSharkLogEntryEditView, { LogEntryEditViewProps } from "./LogEntryEditView";
+import { Layout } from "../../../presentational";
+
+export default { title: "SymptomShark/View/LogEntryEditView", component: SymptomSharkLogEntryEditView, parameters: { layout: 'fullscreen' } };
+let render = (args: LogEntryEditViewProps) => <Layout colorScheme="auto"><SymptomSharkLogEntryEditView {...args} /></Layout>
```

Comment:
Maybe drop the layout here since the view already has one?

Reply:
Ah yep, good catch